config/initializers/rails_admin.rb

```diff
@@ -333,6 +329,10 @@
       field :browse_entries do
         orderable true
         nested_form false
+        inline_add false
+        associated_collection_scope do
+          Proc.new { |scope| BrowseEntry.published }
```

Comment:
Unused block argument - `scope`. You can omit the argument if you don't care about it.